packages/cli-platform-android/src/commands/runAndroid/runOnAllDevices.ts

```diff
@@ -137,6 +138,14 @@ function createInstallError(error: Error & {stderr: string}) {
     message = `Please accept all necessary Android SDK licenses using Android SDK Manager: "${chalk.bold(
       '$ANDROID_HOME/tools/bin/sdkmanager --licenses',
     )}."`;
+  } else if (stderr.includes('requires Java 11 to run')) {
```

Comment:
Can we make this somehow dynamic? This will be more future-proof, and we won't need to change this once next version will released.

Reply:
Can make the error to be just grep of`requires Java` to make it version agnostic and also change the suggestion to remove explicit `jdk 11` instead make it `the compatible version of jdk`